objects/Fan-MadeAccessories.aa8b38/TokenArranger.022907.ttslua

```diff
@@ -237,11 +222,17 @@ function placeTokens()
     -- sort table by value (symbols last if same value)
     table.sort(data, token_value_comparator)
 
+    -- error handling for removal of token arranger
```

Comment:
Seems like this would be better in onDestroy()?  (onDestruct, whatever that method is)

Reply:
That method is getting used. This additional handling is required for the case that the token arranger is getting removed between spawning and positioning the additional tokens.